gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/KubernetesPluginTest.java

```diff
@@ -90,4 +92,15 @@ void getTaskPrecedence_withValidProject_shouldReturnTaskPrecedence() {
         .containsEntry("k8sHelmPush", Collections.singletonList(KubernetesHelmTask.class))
         .containsEntry("k8sHelmLint", Collections.singletonList(KubernetesHelmTask.class));
   }
+
+  @Test
+  void register_shouldRegisterTasks_WithoutReturningTask_kubPlugin() {
+    AbstractJKubePlugin<KubernetesExtension> jkubePlugin = mock(KubernetesPlugin.class, RETURNS_DEEP_STUBS);
+    doNothing().when(jkubePlugin).register(any(),any(),any());
+    // Call the register method
+    jkubePlugin.register(project, "k8sApply", KubernetesApplyTask.class);
+    // Verify that register was called with the expected arguments
```

Comment:
I don't think these comments are adding any value. What do you think?

Reply:
Agree, will remove the comments